connector/ldap/ldap.go

```diff
@@ -107,6 +107,9 @@ type Config struct {
 		IDAttr    string `json:"idAttr"`    // Defaults to "uid"
 		EmailAttr string `json:"emailAttr"` // Defaults to "mail"
 		NameAttr  string `json:"nameAttr"`  // No default.
+
+		// Use a username suffix instead of attribute for email.
```

Comment:
Could you elaborate what this means a little? Maybe something along the lines of _"if this is set, the email claim of the id token will be constructed from ... etc etc"_